package.json

```diff
@@ -20,12 +22,20 @@
     "url": "https://github.com/lostfictions/znv/issues"
   },
   "files": [
-    "!dist/**/*.test.*",
-    "dist/"
+    "dist/",
+    "dist_module/"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist_module/index.js",
+      "require": "./dist/index.js",
+      "types": "./dist_module/index.d.ts"
+    },
+    "./package.json": "./package.json"
```

Comment:
any reason we need to include `package.json` in the export map like this?

Reply:
This would be required if someone wants to read the `package.json` from the package (with json import assertions). However this might be a special case and I removed it for now.